src/components/InfiniteScrollPaginator/InfiniteScroll.tsx

```diff
@@ -0,0 +1,170 @@
+import React, { useCallback, useEffect, useRef } from 'react';
+
+/**
+ * Prevents Chrome hangups
+ * See: https://stackoverflow.com/questions/47524205/random-high-content-download-time-in-chrome/47684257#47684257
+ */
+const mousewheelListener = (event: Event) => {
```

Comment:
would this dom event benefit from a generic event type?

Reply:
generic event type?

Reply:
i mean like something more specific to the event type, like Event<MouseEvent> or something like that